frontend/app/component/error/ErrorModal.tsx

```diff
@@ -0,0 +1,24 @@
+import { ApiError } from "@kiesraad/api";
+import { Button, Modal } from "@kiesraad/ui";
+
+interface ErrorModalProps {
+  error: ApiError;
+}
+
+export function ErrorModal({ error }: ErrorModalProps) {
+  return (
+    <Modal title="Sorry, er ging iets mis" onClose={() => {}}>
+      <div className="content">
+        <p>
+          <strong>
+            Foutcode: <code>{error.code}</code>
+          </strong>
+        </p>
+        <p>{error.error}</p>
+        <Button variant="default" onClick={() => {}}>
+          Melding sluiten
+        </Button>
```

Comment:
This button does not do anything

Reply:
Fixed in 076879b4da23dbabdb80c84c7b3452896459e691